### PR TITLE
DEMO-313: Store Locator

### PR DESCRIPTION
**Pull request by ciastektk**
https://jira.ez.no/browse/DEMO-313

Stores

Maison - Brooklyn

### CODE REVIEW

File: templates/themes/maison/full/store.html.twig [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                </div>
+            </div>
+        </section>
+        <section class="section section--standard my-10">
+            <div class="container">
+                <div class="row mt-">
+                    <div class="col-lg-4 col-md-12">
+                        <h4 class="heading-lg">Find a inspirations and ideas to help you with</h4>
+                        <h5 class="heading-sm mt-4">delivery, assembling, painting, renovation, etc.</h5>
+                        <a href="/inspirations" class="btn btn--lg btn--blue mt-5">Find Inspirations</a>
```
Comment (SylvainGuittard) [inline templates/themes/maison/full/store.html.twig]: Not a big fan of having this hardcoded link. I understand that it's a demo but we should also show best practices.
```suggestion
                        <a href="/inspirations" class="btn btn--lg btn--blue mt-5">{{ 'Find Inspirations' | trans }}</a>
```

File: templates/themes/maison/layout/header.html.twig [hunk 2]
```diff
@@ -22,9 +22,9 @@
                     <a href="#" class="nav-link">Blog</a>
                 </li>
                 <li class="nav-item d-none d-sm-block">
-                    <a href="#" class="nav-link">
+                    <a href="/stores" class="nav-link">
```
Comment (SylvainGuittard) [inline templates/themes/maison/layout/header.html.twig]: Same here, I am not a big fan of hardcoded links.

File: config/packages/views.yml [hunk 3]
```diff
@@ -23,8 +23,30 @@ ezplatform:
                         match:
                             Identifier\ContentType: inspiration
 
+                    store_list:
+                        controller: "ez_query:contentQueryAction"
+                        template: "@ezdesign/full/store_list.html.twig"
+                        match:
+                            Identifier\ContentType: folder
```
[the remark this answers is not in the document]
Comment (ciastektk) [reply config/packages/views.yml]: I'm also not a big fan of this matcher. It was only a temporary approach. Yesterday I wanted to discuss this with @damianz5 but he was on the holiday leave. In my local instance I'm using: Id\Location: 57 matcher.

File: assets/scss/_default.scss [hunk 4]
```diff
@@ -2,12 +2,21 @@
...
-    box-shadow: 0 5px 30px 0 rgba(0,0,0,0.15);
+a {
+    color: $blue-lighter;
+
+    &:hover {
+        text-decoration: none;
+    }
+}
+
+.ezcountry-field {
```
[the remark this answers is not in the document]
Comment (ciastektk) [reply assets/scss/_default.scss]: This style is set for whole project so I think that it is in right place. But after your suggestion I'm wondering on add _ez-fields.scss file and move there this style. In the feature we can also use this file to add styles for ez fields. What do you think?

File: templates/themes/maison/card/store.html.twig [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+                        <p class="font-weight-bold">{{ 'Phone'|trans }}:</p>
+                        <p>{{ contactInfo.cells['phone'] }}</p>
+                    </div>
+                    <div class="card__row">
+                        <p class="font-weight-bold">E-mail:</p>
+                        <p>{{ contactInfo.cells['email'] }}</p>
+                    </div>
+                {% endif %}
+                <div class="card__row">
+                    <p>*
```
[the remark this answers is not in the document]
Comment (ciastektk) [reply templates/themes/maison/card/store.html.twig]: Today we have a meeting with Nikita so I will ask him about this message.

File: templates/themes/maison/full/store_list.html.twig [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+{% block page_head %}
+    {% set title = ez_content_name(content) %}
+    {{ parent() }}
+{% endblock %}
+
+{% block content %}
+    <div class="page">
+        <section class="section banner banner--full-width banner--map">
+            <div class="banner__image">
+                <figure class="ezimage-field">
```
[the remark this answers is not in the document]
Comment (ciastektk) [reply templates/themes/maison/full/store_list.html.twig]: Sorry, i forgot to remove it. Today I will ask Nikita about banners. I think that every baner should be taken from Content object, not hardcoded in template.

File: config/packages/views.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (bdunogier) [reply config/packages/views.yml]: Did you consider a custom content type for stores ? You would not need an id based override then.

File: templates/themes/maison/full/store.html.twig [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="section section mt-8">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-6">
+                        <h2 class="heading-sm">Store details</h2>
+                        <div class="mt-3">{{ ez_render_field(content, 'description') }}</div>
```
[the remark this answers is not in the document]
Comment (ciastektk) [reply templates/themes/maison/full/store.html.twig]: ```intro``` shouldn't be there. I put it only for test purposes to have more content but I forgot to delete it.

File: config/packages/views.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (SylvainGuittard) [reply config/packages/views.yml]: `store_list` was my first idea when I saw this matcher. I told Nikita about it. Not sure if he made progress about that.

File: assets/scss/_default.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (SylvainGuittard) [reply assets/scss/_default.scss]: It might be a good idea to move it to `_ez-fields.scss`.

**Comment by ciastektk** [comment]
@SylvainGuittard, thanks for your review! I made changes requested by you.

Don't worry about hardcoded links. I don't like it also but this is only temporary approach.

File: config/packages/views.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (ciastektk) [reply config/packages/views.yml]: I agree with you. This should be content type store list

**Pull request merged**